errors.go

```diff
@@ -109,7 +109,7 @@ func (e errorFrame) Message() string {
 }
 
 func (e errorFrame) Error() string {
-	return e.Message()
+	return "code=" + fmt.Sprintf("%x", e.Code()) + "; message=" + e.Message()
```

Comment:
Is a case with nil code possible? If yes, will this result in error or it would just read as `code=nil`?

Reply:
nil code is not possible, If any improper code is send by the server it will be handled here : https://github.com/yugabyte/gocql/blob/master/frame.go#L698